roles/ceph-common/tasks/main.yml

```diff
@@ -67,6 +78,12 @@
     is_ceph_infernalis={{ (ceph_stable and ceph_stable_release not in ceph_stable_releases) or (ceph_dev) or (ceph_stable_rh_storage and (rh_storage_version.stdout | version_compare('0.94', '>'))) }}
 
 - set_fact:
+    is_ceph_infernalis=True
```

Comment:
I vote that we keep this PR as-is. We are introducing ceph-ansible support for RHCS with RHCS 2, which is Jewel-based. Simply setting this fact to "True" here keeps things simple.

Honestly I'd rather we ripped out hammer support (including the `is_ceph_infernalis` variable) throughout the tree entirely, to drastically simplify the codebase. But maybe that will need to happen in ceph-ansible 2.x.